Classes/Backend/SolrDocHeader.php

```diff
@@ -32,12 +32,6 @@ public function __construct(
 
     public function renderDocHeader(): void
     {
-        // initialize UriBuilder
-        if (!($this->uriBuilder instanceof UriBuilder)) {
-            $this->uriBuilder = GeneralUtility::makeInstance(UriBuilder::class);
-        }
-        $this->uriBuilder->setRequest($this->request);
```

Comment:
I understand the 3 lines above, but why you have removed this line?

Reply:
Because the request is already set with dependency injection.

Reply:
Can you please show me the line where the Request class was included into UriBuilder via DependencyInjection?